internal/app/models/userModel/userModel.go

```diff
@@ -0,0 +1,142 @@
+package userModel
```

Comment:
Can we create some tests for these functions? (or if there already are some, can you direct me to them, I'm fried)

Reply:
I think the integration tests should cover all the model logic, since they're not unit testable.

Reply:
I just put each integration test under its respective handler, since that's where the test starts.

Reply:
Hmm now I think about it, I'm considering merging each service and handler into one package

Reply:
Yeah you could, it's not really necessary to separate the handler if it only have one fn.